js/app/Visualization/Animation/CartoDBAnimation.js

```diff
@@ -86,7 +86,9 @@ define([
       LoadingInfo.main.add(url, true);
 
       self.sql.execute(
-        (  self.layer.getSubLayer(layerIndex).getSQL()
+        (  'select * from ('
```

Comment:
@redhog on CartoDB map you can select which attributes to be displayed and in which order. I wonder if there is a way of querying those.